zerver/models.py

```diff
@@ -93,7 +93,7 @@ def get_display_recipient_remote_cache(recipient_id, recipient_type, recipient_t
     # We don't really care what the ordering is, just that it's deterministic.
     user_profile_list = (UserProfile.objects.filter(subscription__recipient_id=recipient_id)
                                             .select_related()
-                                            .order_by('email'))
+                                            .order_by('id'))
```

Comment:
The story here is that we want the ordering of users in the naming of the group huddle message to be the same as what is displayed on the left-hand sidebar of Zulip to be consistent with what the group huddle names look like on Zulip and in the missed message emails.  The javascript in static/js/people.js sorts by id, so since the comment says we don't really care what the ordering here is as long as it's deterministic, I changed this ordering to match that one.